### PR TITLE
Dropped `name` field from API requests (#171)

### PR DESCRIPTION
**Pull request by jkohen**
This is a cherry-pick of #171.

### CODE REVIEW

**Review by igorpeshansky — CHANGES_REQUESTED** [review]
Let's get #171 in first, and then use this one to cherry-pick the result into `stackdriver-agent-5.5.2`?

File: src/write_gcm.c [hunk 1]
```diff
@@ -2867,29 +2867,17 @@ static void wg_json_RFC3339Timestamp(json_ctx_t *jc, cdtime_t time_stamp);
 //   string collectd_version = 3;
 //   repeated CollectdPayload collectd_payloads = 4;
 // }
-static int wg_json_CreateCollectdTimeseriesRequest(_Bool pretty,
-    const monitored_resource_t *monitored_resource,
-    const wg_payload_t *head, const wg_payload_t **new_head,
-    char **json) {
```
Comment (igorpeshansky) [inline src/write_gcm.c]: Given that this is in the 5.5.2 branch, can we please keep the original formatting? Also below.

**Review by igorpeshansky — CHANGES_REQUESTED** [review]
One formatting request. Also, can we please update the PR description to match, and mention that it's cherry-picking #171?

File: src/write_gcm.c (hunk 1 above)
Comment (jkohen) [reply src/write_gcm.c]: Done.

**Review by igorpeshansky — APPROVED** [review]
LGTM :shipit:

**Pull request merged**